Automattic/Automattic-Tracks-Android
# Sentry Performance configuration

## Description

This PR adds support for the Sentry Performance Monitoring feature for clients with an anti-bikeshading approach. We expose only limited portion of easy to understand API.

As our needs will grow in the future, we'll expose more needed API. But for now it's completely sufficient. 

### How to test
1. Have a test Sentry project with Performance Monitoring enabled.
2. Launch the test app
3. Click on `Measure performance of sample HTTP fetch`
4. Make sure that after a while you can see a report similar to:

<img width="2536" alt="image" src="https://user-images.githubusercontent.com/5845095/182347863-b5aa291a-15d6-4dff-947f-7795835a56ee.png">
